nnc/Store.swift

```diff
@@ -441,6 +538,8 @@ extension DynamicGraph {
           codec = .zip
         case 0xf7217:
           codec = .fpzip
+        case 0x511:
+          codec = .ezm8
```

Comment:
why do we encode the codec if we also specify at runtime with the codecs: parameter? shouldn't we not have to specify the codecs: parameter during reading?